pkg/deployer/deployer.go

```diff
@@ -82,6 +83,7 @@ func (e ethereumDeployer) DeployContract(contractInfo *ContractInfo, containerIn
 	defer os.RemoveAll(outputFile.Name())
 
 	cmd := exec.Command("node", e.deployerPath, "-c", configFile.Name(), "-o", outputFile.Name(), contractInfo.ContractPath)
+	e.logger.Debug(fmt.Sprintf("%++v", cmd))
```

Comment:
this is heplful to see what the cmd looks like before it gets run. make sure all the args are present.